app/renderer/js/utils/domain-util.js

```diff
@@ -103,11 +112,14 @@ class DomainUtil {
 
 	// ignoreCerts parameter helps in fetching server icon and
 	// other server details when user chooses to ignore certificate warnings
-	checkDomain(domain, ignoreCerts = false, silent = false) {
+	checkDomain(domain, {ignoreCerts = false, silent = false, override = false} = {}) {
 		if (!silent && this.duplicateDomain(domain)) {
 			// Do not check duplicate in silent mode
 			return Promise.reject('This server has been added.');
 		}
+		if (EnterpriseUtil.isAdminOnly('presetOrganizations') && override === false) {
+			return Promise.reject('Adding organizations is a restricted operation. Please contact your system administrator.');
```

Comment:
I don't think we want to restrict users from adding a new organization. The most we should restrict them to do is removing a preset org (and even that I feel like should be a new setting).

cc @akashnimare

Reply:
Okay, I'll wait for more discussion on this.

Reply:
Agreed.